### ISSUE TITLE
In yolo_head.py 410 line. Why set reg_weight=5.0? 

### ISSUE THREAD

**Issue by ZhenxinYUAN** [Megvii-BaseDetection/YOLOX#609]
Hi, I have a question about loss compute.
In yolo_head.py 410 line.

**reg_weight = 5.0**
loss = reg_weight * loss_iou + loss_obj + loss_cls +loss_l1

Why set reg_weight=5.0? and  What does reg_weight mean?

**Comment by Joker316701882** [comment]
@ZhenxinYUAN It is an **empirical** number for controlling the loss weight of the iou_loss.

**Comment by ZhenxinYUAN** [comment]
Thank you for your recovery.
This value is set to 5 according to what?
Is it just experience or is it something else?

**Comment by Joker316701882** [comment]
@ZhenxinYUAN Grid search. You can use 4 or 6, doesn't matter that much. I believe this is not the point that deserves your attention.

**Comment by ZhenxinYUAN** [comment]
Ok, thanks for your reply!

**Comment by ZhenxinYUAN** [comment]
@Joker316701882  Ok, thanks for your reply!
